### ISSUE TITLE
Feature request: (optionally) compile imported code

### ISSUE THREAD

**Issue by zdateh** [intersystems/git-source-control#335]
When switching to a different branch, it is necessary to do an "Import All" via the Studio git menu. This works as advertised, but for the imported code to be executable it needs to be compiled again, otherwise you'll still be running the previous branch. For this we go into the command line and do:

```
do $SYSTEM.OBJ.CompileAll() ; Compiles all classes
do ##class(%Library.Routine).CompileAll() ; Compiles all routines
```

Which is fine, but ideally the Import would automate this. Perhaps a separate 'Import All and Compile' option? Or a Yes/No prompt after Importing? Or perhaps a global setting to control compilation?

Thanks,
Chris

**Comment by isc-tleavitt** [comment]
This is a great idea. Thank you!

**Comment by isc-etamarch** [comment]
@zdateh This issue was addressed by #362 and a fix is available in version 2.4.0

In the settings page there is a compile on import slider that can be checked, which will compile all items when imported.
